packages/core/components/FileList/useFileAccessContextMenu.ts

```diff
@@ -19,6 +21,46 @@ export default function useFileAccessContextMenu(filters?: FileFilter[], onDismi
     const userSelectedApplications = useSelector(interaction.selectors.getUserSelectedApplications);
     const { executionEnvService } = useSelector(interaction.selectors.getPlatformDependentServices);
 
+    const [hasPlateBarCode, setHasPlateBarCode] = React.useState(false);
+    let plateUILink: string | undefined = undefined;
+
+    fileSelection.fetchAllDetails().then((fileDetails) => {
+        console.log(fileDetails);
```

Comment:
Generally we remove logs like this

Reply:
was fixed in [my prev. commit](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/47/commits/0b411a367e1e160e56602ee1f1e5466ef5ceb085)